cmd/program/program.go

```diff
@@ -571,4 +669,40 @@ func (p *Project) CreateFileWithInjection(pathToCreate string, projectPath strin
 	}
 
 	return nil
-}
\ No newline at end of file
+}
+
+func (p *Project) CreateTemplateRoutes() {
+	placeHolder := ""
+	if p.AdvancedOptions["AddHTMXTempl"] {
+		placeHolder += string(p.FrameworkMap[p.ProjectType].templater.HtmxTemplRoutes())
+	}
+
+	phTmpl, err := template.New("routes").Parse(placeHolder)
+	if err != nil {
+		log.Fatal(err)
+	}
+	var phBuffer bytes.Buffer
+	err = phTmpl.Execute(&phBuffer, p)
+	if err != nil {
+		log.Fatal(err)
+	}
+	p.AdvancedTemplates.TemplateRoutes = template.HTML(phBuffer.String())
+}
+
+func (p *Project) CreateTemplateImports() {
```

Comment:
This func looks almost identical to CreateTemplateRoutes.
Perhaps isntead of using two diff funcs for two separate purposes, can we make one func that does the same purpose for 2 different things?